single-profile.php

```diff
@@ -66,50 +66,49 @@
+								}
+								?>
+								<div class="bold profile-contacts"><a href="<?php echo strpos( $link['link'], 'mailto' ) !== false ? esc_url( 'mailto:' . antispambot( str_replace( 'mailto:', '', $link['link'] ) ) ) : esc_url( $link['link'] ); ?>">
+									<img src="<?php echo esc_url( $img ); ?>" alt="">
+									<?php echo esc_html( $link['title'] ); ?>
+								</a></div>
+							</div>
+						<?php endforeach; ?>
+					<?php endif; ?>
+					<?php if ( ! empty( $connected_user ) ) : ?>
 						<?php
-						$img = "";
-						if ( is_int(strpos($link['link'],'meta.wikimedia.org') ) ) {
-							$img = get_template_directory_uri() . "/assets/src/svg/globe.svg";
-						}
-						if ( is_int(strpos($link['link'],'mailto')) ) {
-							$img = get_template_directory_uri() . "/assets/src/svg/email.svg";
-						}
-						if ( is_int(strpos($link['link'],'wikipedia.org')) ) {
-							$img = get_template_directory_uri() . "/assets/src/svg/individual/wikipedia.svg";
-						}
-						if ( is_int(strpos($link['link'],'/news/')) ) {
-							$img = get_template_directory_uri() . "/assets/src/svg/individual/wikimedia-blue.svg";
-						}
+							$authorimg = get_template_directory_uri() . "/assets/src/svg/edit-ltr.svg";
+							if ( is_rtl() ) {
+								$authorimg = get_template_directory_uri() . "/assets/src/svg/edit-rtl.svg";
+							}
+							$authorlink = wmf_get_author_link( $connected_user );
+							$authorlinkcopy = sprintf( /* translators: 1. post title */ __( 'Posts by %s', 'shiro' ), get_the_title() );
 						?>
-						<div class="bold profile-contacts"><a href="<?php echo strpos( $link['link'], 'mailto' ) !== false ? esc_url( 'mailto:' . antispambot( str_replace( 'mailto:', '', $link['link'] ) ) ) : esc_url( $link['link'] ); ?>">
-							<img src="<?php echo esc_url($img); ?>" alt="">
-							<?php echo esc_html( $link['title'] ); ?>
-                        </a></div>
-					</div>
-					<?php endforeach; ?>
-				<?php endif; ?>
-				<?php if ( ! empty( $connected_user ) ) : ?>
-                    <?php
-						$authorimg = get_template_directory_uri() . "/assets/src/svg/edit-ltr.svg";
-						if ( is_rtl() ) {
-							$authorimg = get_template_directory_uri() . "/assets/src/svg/edit-rtl.svg";
-						}
-                        $authorlink = wmf_get_author_link( $connected_user );
-                        $authorlinkcopy = sprintf( /* translators: 1. post title */ __( 'Posts by %s', 'shiro' ), get_the_title() );
-                    ?>
-                    <div class="link-list mar-right">
-                    <div class="bold profile-contacts"><a href="/news/author/<?php echo esc_attr( $authorlink ); ?>">
-							<img src="<?php echo esc_url($authorimg); ?>" alt="">
-							<?php echo esc_html( $authorlinkcopy ); ?>
-						</a></div>
-                    </div>
-				<?php endif; ?>
+						<div class="link-list mar-right">
+						<div class="bold profile-contacts"><a href="/news/author/<?php echo esc_attr( $authorlink ); ?>">
```

Comment:
Strange that this isn't using get_author_posts_link. But it looks like this duplicates the existing functionality, so it should be fine.